gap/main.g

```diff
@@ -580,9 +580,17 @@ Chief := function( galoisField,mat,a,b,IsHPC,withTrafo )
         D := TransposedMat( GAUSS_RRF( galoisField,X,
             TransposedMat( GAUSS_CEX( galoisField,v,D )[1] ),v ) );
     fi;
-    
 
+    
     heads := GAUSS_createHeads(v, w, DimensionsMat(mat)[2]);
+
+    if verify and not IsMatrixInRREF(Concatenation(-B, D) * mat) then
+        Error("Result verification failed! Please report this bug to ",
+              "https://github.com/lbfm-rwth/GaussPar#contact");
+    fi;
+
+    result := rec(vectors := -C, pivotrows := v, pivotcols := w, rank := rank,
```

Comment:
Why do you add this line? I can't find where it is used.

Reply:
Okay, I found out where `result` came from! It's interesting, that the tests work...

Reply:
Ah, I think that's my mistake. I've split up some commit and I guess this line somehow snuck in from #177. :) I'll make sure to clean that up when I resolve the merge conflict with master.

Reply:
I guess the tests worked because we didn't use `result` at all? I have now rebased this branch onto master and this part should now make a lot more sense.